src/components/validering/validateFnrSingle.tsx

```diff
@@ -0,0 +1,15 @@
+import fnrvalidator from '@navikt/fnrvalidator';
+import { Ansatt } from '../../data/types/sporenstreksTypes';
+
+export const validateFnrSingle = (fnr?: string): string | undefined => {
```

Comment:
Jeg kan ikke se relevansen for å slutte på "single"...? Litt misvisende

Reply:
Den validerer bare et fnr, i motsetning til den som bare heter validateFnr som tar med alle fnr'ene. 
Skal tenke litt på det...

Reply:
Ah. Da er det den andre metoden som burde navn egentlig. Se bort fra kommentaren.

Reply:
Navnet den om så navnet viser litt bedre hva den gjør.